_posts/2018-11-20-keeping-dependencies-updated.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: epic
+title: Keeping Arty's dependencies up to date
+date: 2018-11-20
+author: justin
+categories: [best practices, dependencies, javascript, node, roads and bridges]
+---
+
+Hey everyone! I'm Justin, a senior engineer here at Artsy. I've been here for about 6 months and I'm a bit overdue
+on my first blog post. This will be one of a series of posts I'm dubbing _roads and bridges_ (thanks Orta)
+describing infrastructure and tooling work here at Artsy.
+
```

Comment:
We can make this a proper series (and retroactively look for other posts in a similar vein, I bet we have a. few)

Reply:
Want me to just remove this part then?

Reply:
nah, this is good priming